cmd/kubelet/app/options/options.go

```diff
@@ -60,74 +51,16 @@ type KubeletServer struct {
 
 // NewKubeletServer will create a new KubeletServer with default values.
 func NewKubeletServer() *KubeletServer {
+	config := componentconfig.KubeletConfiguration{}
```

Comment:
@liggitt - fyi, still reviewing this, but it looks like defaulting has been moved here....